backend/app/services/knowledge/knowledge_service.py

```
@@ -122,6 +129,31 @@ class ActiveDocumentTextStats:
+        return user
+
+    @staticmethod
+    def _get_knowledge_base_record(
+        db: Session, knowledge_base_id: int
+    ) -> Optional[Kind]:
+        return (
+            db.query(Kind)
+            .filter(
+                Kind.id == knowledge_base_id,
+                Kind.kind == "KnowledgeBase",
+                Kind.is_active == True,
```

Ruff (0.15.7)

[error] Avoid equality comparisons to `True`; use `Kind.is_active:` for truth checks

Replace with `Kind.is_active`

(E712)